fire/SFMainMod.F90

```diff
@@ -916,21 +923,49 @@ subroutine  crown_damage ( currentSite )
 
           do while(associated(currentCohort))  
              currentCohort%fraction_crown_burned = 0.0_r8
+             currentCohort%passive_crown_FI      = 0.0_r8  !critical fire intensity for passive crown fire
+             currentCohort%crown_fire_flg        = 0       !flag for passvie crown fire ignition
+             
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
+                
+                ! Calculate clear branch bole height at base of crown
+                ! inst%crown = crown_depth_frac (PFT)
+                height_cbb = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)
+                
+                ! Evaluate for passive crown fire ignition
+                if (EDPftvarcon_inst%crown_fire(currentCohort%pft) == 1) then
+                   
+                   ! Crown fuel ignition potential, EQ 8 Bessie and Johnson 1995
+                   currentCohort%passive_crown_FI = (0.01 * height_cbb * SF_val_crown_ignition_energy)**1.5
+
+                   if (currentCohort%passive_crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
+                      
+                      ! Initiation of passive crown fire, EQ 14 Bessie and Johnson 1995
+                      currentCohort%ignite_crown = currentPatch%FI/currentCohort%passive_crown_FI
+                      
+                      if (currentCohort%ignite_crown > 1.0_r8) then
```

Comment:
Minor correction. Looking at the article again I realized that this should be >=

```suggestion
                      if (currentCohort%ignite_crown >= 1.0_r8) then
```

This is the sentence in the paper that explains:
![image](https://user-images.githubusercontent.com/35322900/64462569-be303880-d0b5-11e9-8626-804e9add8373.png)